package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "memeing-maker",
+  "version": "1.0.0",
+  "description": "Making Memeing Maker: A game for creating and sharing meaningful memes",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/limitless-harmony/memeing-maker.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/limitless-harmony/memeing-maker/issues"
+  },
+  "homepage": "https://github.com/limitless-harmony/memeing-maker#readme",
+  "dependencies": {
+    "bluebird": "^3.5.3",
+    "dotenv": "^6.2.0",
+    "express": "^4.16.4",
+    "mongoose": "^5.4.18"
+  },
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-stage-0": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "chai": "^4.2.0",
+    "chai-http": "^4.2.0",
+    "cors": "^2.8.5",
```

Comment:
@Avuidrauxs 
Shouldn't this be a dependency instead of a devDependency?

Reply:
Oh yeah I needed a second opinion on this. @Veraclins